tools/tests/cordova-hcp.js

```diff
@@ -39,3 +39,50 @@ selftest.define(
 
     run.stop();
 });
+
+selftest.define(
+  "cordova CORDOVA_COMPATIBILITY_VERSION_* works", ["cordova", "slow"], function () {
+    var s = new Sandbox();
+    var run;
+
+    var androidCompatibilityVersion = '2.0';
+
+    // Override the compatibility version for android with CORDOVA_COMPATIBILITY_VERSION_ANDROID.
+    s.env.CORDOVA_COMPATIBILITY_VERSION_ANDROID = androidCompatibilityVersion;
+
+    s.createApp("myapp", "standard-app");
+    s.cd("myapp");
+
+    var platforms = s.read(".meteor/platforms");
+    s.write(".meteor/platforms", platforms + "\nandroid\n");
+
+    run = s.run("run", "android", "--mobile-server", "example.com");
+    run.waitSecs(30);
+    run.match("Started your app");
+
+    var result = JSON.parse(httpHelpers.getUrl(
+        "http://localhost:3000/__cordova/manifest.json"));
```

Comment:
I've just used the same approach that was already used https://github.com/meteor/meteor/blob/devel/tools/tests/cordova-hcp.js#L34
`meteor test-packages --port 3500` this is not the case because this is `self-test` and you can not pass a port to it, right?